content/spin/v2/quickstart.md

```diff
@@ -282,21 +279,23 @@ The additional `spin.toml` file is the manifest file, which tells Spin what even
 <!-- @nocpy -->
 
 ```toml
-spin_manifest_version = "1"
+spin_manifest_version = "2"
 authors = ["You <you@yourself.net>"]
```

Comment:
I don't know if you care about exactly tracking the generated manifest but if you do, these fields are now ordered differently (so that 'name' is the most prominent):

```
spin_manifest_version = 2

[application]
name = "v2vanilla"
version = "0.1.0"
authors = ["itowlson <ivan.towlson@fermyon.com>"]
description = ""
```